controllers/idler/idler_controller.go

```diff
@@ -297,7 +285,7 @@ func (r *Reconciler) notify(ctx context.Context, idler *toolchainv1alpha1.Idler,
 		logger.Error(err, "failed to create Notification")
 		if err = r.setStatusIdlerNotificationCreationFailed(ctx, idler, err.Error()); err != nil {
 			logger.Error(err, "failed to set status IdlerNotificationCreationFailed")
-		} // not returning error to continue tracking remaining pods
+		} // not returning error to continue processing remaining pods
```

Comment:
would it make sense to return error/list of error at the end?

Reply:
in this particular case, not really. Reconciling again wouldn't help with sending the notification again